src/main/cpp/drivers/aaudio_io.cpp

```diff
@@ -162,7 +167,7 @@ void AAudio_IO::setRecordingDeviceId( int32_t deviceId ) {
  */
 AAudioStreamBuilder* AAudio_IO::createStreamBuilder() {
     AAudioStreamBuilder* builder = nullptr;
-    aaudio_result_t result = AAudio_createStreamBuilder( &builder );
+    aaudio_result_t result = static_cast<aaudio_result_t>(AAudio_createStreamBuilder( &builder ));
     if ( result != AAUDIO_OK && !builder ) {
```

Comment:
With the runtime loaded library, we need to cast the _int_32_t_ values to the enumerations defined in _aaudio.h_, this is the only explicit awareness within MWEngine of AAudio's inclusion being runtime or within the compilation unit.